app/models/infra_conversion_job.rb

```diff
@@ -1,4 +1,7 @@
 class InfraConversionJob < Job
+  # Override default timeout to allow large disks VM migration
+  DEFAULT_TIMEOUT = 36.hours
```

Comment:
Prefer overriding the `self.current_job_timeout` method instead of redefining the constant